micromasters/utils.py

```diff
@@ -19,6 +19,14 @@ def webpack_dev_server_url(request):
     return 'http://{}:{}'.format(webpack_dev_server_host(request), settings.WEBPACK_DEV_SERVER_PORT)
 
 
+def remove_key(dictionary, key):
+    """
+    Helper method to remove a key from a dict and return the dict
+    """
+    del dictionary[key]
+    return dictionary
```

Comment:
actually, returning the dictionary here is useless, because the `del` is working in the same object passed via arg from the parent. And this means that the function itself does not have a real meaning... Unless I am missing something